test/e2e/flask/multichain-api/invoke-method.spec.ts

```diff
@@ -0,0 +1,274 @@
+import { strict as assert } from 'assert';
+import {
+  ACCOUNT_1,
+  ACCOUNT_2,
+  largeDelayMs,
+  WINDOW_TITLES,
+  withFixtures,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import {
+  initCreateSessionScopes,
+  DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+  openMultichainDappAndConnectWalletWithExternallyConnectable,
+  addAccountInWalletAndAuthorize,
+} from './testHelpers';
+
+describe('Multichain API', function () {
+  const GANACHE_SCOPES = ['eip155:1337', 'eip155:1338', 'eip155:1000'];
+  const ACCOUNTS = [ACCOUNT_1, ACCOUNT_2];
+  const DEFAULT_INITIAL_BALANCE_HEX = '0x15af1d78b58c40000';
+
+  describe('Calling `wallet_invokeMethod` on the same dapp across three different connected chains', function () {
+    describe('Read operations: calling different methods on each connected scope', function () {
+      it('Should match selected method to the expected output', async function () {
```

Comment:
maybe not strictly necessary but perhaps we could add a test where we invoke each method and let the confirmations pop up without confirming them until all are up. The reason for this is to test how the Queueing is functioning - atleast until we remove the QueuedRequestController.

Reply:
oh wait nvm this is what you're doing on the write operations